core/src/test/java/org/apache/iceberg/TestBase.java

```diff
@@ -63,7 +63,7 @@ public class TestBase {
   public static final PartitionSpec SPEC =
       PartitionSpec.builderFor(SCHEMA).bucket("data", BUCKETS_NUMBER).build();
 
-  static final DataFile FILE_A =
+  public static final DataFile FILE_A =
```

Comment:
Why did these need to change?

Reply:
I see, this is used in the tests below. Could we just have the tests which use this to extend `TestBase?

Reply:
that actually makes more sense, can try this out

Reply:
extending `TestBase` means that the subclass is now a parameterized test, which I don't think we actually need

Reply:
Good point, we technically extend TestBase in sort order parser tests but we just force a single version in the parameter. We could either do that, or honestly just create the data file again, I think that bit of duplication is nbd.

Reply:
@amogh-jahagirdar Just curious what is the concern/issue with the change? I think I can try adding this variable directly to tests but will need to do this in multiple places. Rather I think we can keep this change as the `TestBase` class seems to have other public variables, this seems to be clean to me.